### ISSUE TITLE
Ability to apply different colors for dark and light system theme

### ISSUE THREAD

**Issue by japanese-goblinn** [dbrgn/tealdeer#281]
This is more `macOS`/`Windows` specific request because i don't really know is there a common API for this on linux but it'll be a nice to have ability to apply different color depending on current theme or even better - a different configs

**Comment by japanese-goblinn** [comment]
Also I'm learing Rust know and really like the idea to implement this myself and create PR if you don't mind c:

**Comment by dbrgn** [comment]
Is there an API to determine the current terminal theme on macOS and Windows?

**Comment by japanese-goblinn** [comment]
I was talking about system theme not a terminal theme. I think I came with even better idea than implementing interactions with vague API. You just need to implement ability to pass a path to config directly to `tldr` so it'll be`tldr --config-path="$HOME/config.toml" and with a little wrapper around users can implement this on there's platforms. For example in macOS it'll be something like this:

```shell
function _is_macos_dark() {
  defaults read -globalDomain AppleInterfaceStyle &>/dev/null && true || false
}

function my_tldr() {
	if _is_macos_dark; then
		tldr --config-path="$HOME/light_config.toml"
  	else
		tldr --config-path="$HOME/dark_config.toml"  		
  	fi
}
```

**Comment by dbrgn** [comment]
@japanese-goblinn yes, that looks like a good idea. Configurable config is being developed here: #141 Would that solve your issue?

**Comment by japanese-goblinn** [comment]
Yes, I think it will. Closing this one
